MapboxNavigationTests/Fixtures/vanish_point_test.json

```diff
@@ -0,0 +1 @@
type":"text","abbr":"2nd St","abbr_priority":0}],"type":"turn","modifier":"left"}}],"driving_side":"right","weight":81.8,"intersections":[{"location":[-122.52753,37.972075],"bearings":[90,180,270],"entry":[false,true,true],"in":0,"out":1}]},{"distance":105.0,"duration":7.5,"geometry":"shqlgAxznuhF|FeiA","name":"2nd Street","mode":"driving","maneuver":{"location":[-122.527677,37.971098],"bearing_before":185.0,"bearing_after":97.0,"instruction":"Turn left onto 2nd Street","type":"turn","modifier":"left"},"voiceInstructions":[{"distanceAlongGeometry":70.0,"announcement":"You have arrived at your destination, on the right","ssmlAnnouncement":"\u003cspeak\u003e\u003camazon:effect name\u003d\"drc\"\u003e\u003cprosody rate\u003d\"1.08\"\u003eYou have arrived at your destination, on the right\u003c/prosody\u003e\u003c/amazon:effect\u003e\u003c/speak\u003e"}],"bannerInstructions":[{"distanceAlongGeometry":105.0,"primary":{"text":"You will arrive","components":[{"text":"You will arrive","type":"text"}],"type":"arrive","modifier":"right"}},{"distanceAlongGeometry":70.0,"primary":{"text":"You have arrived","components":[{"text":"You have arrived","type":"text"}],"type":"arrive","modifier":"right"}}],"driving_side":"right","weight":7.5,"intersections":[{"location":[-122.527677,37.971098],"bearings":[0,105,270],"entry":[false,true,false],"in":0,"out":1}]},{"distance":0.0,"duration":0.0,"geometry":"u`qlgArpluhF","name":"2nd Street","mode":"driving","maneuver":{"location":[-122.52649,37.970971],"bearing_before":98.0,"bearing_after":0.0,"instruction":"You have arrived at your destination, on the right","type":"arrive","modifier":"right"},"voiceInstructions":[],"bannerInstructions":[],"driving_side":"right","weight":0.0,"intersections":[{"location":[-122.52649,37.970971],"bearings":[278],"entry":[true],"in":0}]}],"annotation":{"distance":[49.34180914849393,33.05802569090612,10.689795908138624,39.59839199650681,34.80992351675273,14.209672171473654,55.332462814681335,8.615785848371143,40.91659557025385,5.914738766868038,52.97482177741399,85.20461216655733,12.501699782507817,35.42252405667138,29.311743333674485,35.66534898263901,25.758372926868166,44.32194001964289,22.517423451854935,28.451254156899267,40.20997808786729,30.687302775025532,38.532552346674045,105.03289645548367],"congestion":["low","unknown","unknown","low","unknown","unknown","heavy","low","low","unknown","low","heavy","low","low","low","low","low","low","low","low","unknown","unknown","unknown","low"]}}],"routeOptions":{"baseUrl":"https://api.mapbox.com","user":"mapbox","profile":"driving-traffic","coordinates":[[-122.5237734,37.9753973],[-122.5264995,37.9709171]],"alternatives":true,"language":"en","continue_straight":false,"roundabout_exits":false,"geometries":"polyline6","overview":"full","steps":true,"annotations":"congestion,distance","voice_instructions":true,"banner_instructions":true,"voice_units":"imperial","access_token":"someToken","uuid":"ckd9ao6hl13a97ars2byaymo7"},"voiceLocale":"en-US"}
```

Comment:
Do we really need to add test data twice in both TestHelper and MapboxNavigationTests?

Reply:
I'm not sure. But if I remove the file from the  MapboxNavigationTests, the Fixture could not find it.

Reply:
There is no need to add `vanish_point_test.json` to MapboxNavigationTests I think. You can remove another copy and make sure it's only added to TestHelper target like on screenshot. Can you also rename it to be: `route-for-vanishing-route-line.json`? 
<img width="256" alt="Screen Shot 2020-12-03 at 10 38 17 AM" src="https://user-images.githubusercontent.com/1496498/101073135-dc544500-3553-11eb-8e35-6935dc10b227.png">

Reply:
Thanks for the suggestion. I'll work on it